ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -1699,3 +1699,31 @@ FCU_DSP_DFVSC_DTM_RTM:
   - DTM
   - RTM
 
+
+FCU_DXZTC_DFSS:
+  guid: "95515cb5-eb97-4acf-9be2-a792c8cf99e4"
+  description: "FCU with SS, discharge temp sensors, zone control and sensor, filter change"
+  is_canonical: true
+  implements:
+  - FCU
+  - DXZTC
+  - DFSS
+  opt_uses:
+  - failed_alarm
```

Comment:
Please remove this, it is not specific enough (what piece of the FCU has failed?). If you are just trying to model if the FCU has failed to communicate please note that the field failed_communciation_alarm already exists on the FCU type

Reply:
the pointlist indicate that this is a general alarm for the device, sort of a catch-all that something in the unit has failed (other than the alarms already defined optional).